requirements/common.txt

```diff
@@ -18,4 +18,5 @@ Pillow==3.4.2
 psycopg2==2.7.3.2
 PyYaml==4.2b1
 proc==0.10.1
+responses==0.5.1
```

Comment:
Nice. This is what we were looking for. But do you think we need this in prod config? If not, then please put it in dev.txt

Reply:
@deshraj I'm not really sure if we need this in prod. If 'travis' uses the prod build for testing then we will need it for sure. If in the prod build, we were to remove the responses codes and put in the actual webhook then it might work for dev.txt only

Reply:
We actually don't need this in prod. Feel free to move it to dev.py